### DISCUSSION TITLE
Works or Projects on Traffic simulation

### DISCUSSION THREAD

**Discussion by latchukarthick98** [FLAMEGPU/FLAMEGPU2#879] [General]
Hello All,
I have been studying FLAME GPU and its applications. I wonder if there are any traffic simulation projects that uses FLAME GPU. I couldn't find anything on my one. Your help is appreciated.

Thanks,
Lakshman Karthik Ramkumar.

**Comment by ptheywood** [comment]
We've previously used FLAME GPU 1.5 for some road network miscrosimualtion work, described in https://doi.org/10.1016/j.simpat.2017.11.002.
It was also presented at GTC in 2019, as part of Session 9387, with [slides](https://ptheywood.uk/download/presentations/heywood-gtc2019-presentation.pdf) and a [recording available (login required)](https://developer.nvidia.com/gtc/2019/video/s9387).

Unfortunately the code for this model is not available. 

The communication pattern used within that work (where messages are associated with the graph representing the road network) has not yet been implemented for FLAME GPU 2, but implementation details are in the process of being worked out / discussed.
